core/templates/dev/head/pages/exploration_editor/editor_tab/state_editor_solution.html

```diff
@@ -22,16 +22,10 @@
             </div>
             <a ng-click="toggleInlineSolutionEditorIsActive()"
                class="oppia-rule-tab protractor-test-oppia-solution-tab">
-              <div class="oppia-response-header-block">
-                <div class="oppia-response-header">
-                  <span angular-html-bind="getSolutionSummary()"></span>
-                  <span class="pull-right oppia-transition-200"
-                        ng-click="deleteSolution($event)"
-                        style="cursor: pointer; opacity: 0.5;">
-                    <i class="material-icons md-18">&#xE5CD;</i>
-                  </span>
-                </div>
-              </div>
+               <response-header summary="getSolutionSummary()"
```

Comment:
Deindent this and the following lines by 1. (It should be indented by 2 from the outer scope.)